test/unit/format/vcf_input_test.cpp

```diff
@@ -20,10 +20,8 @@ void field_types_bcf_style()
 {
     std::istringstream istr{std::string{example_from_spec}};
 
-    using record_t =
-      bio::detail::record_from_typelist<std::remove_cvref_t<decltype(bio::var_io::default_field_ids)>,
-                                        std::remove_cvref_t<decltype(bio::var_io::field_types_bcf_style<own>)>>;
-    ;
+    using record_t = bio::record<std::remove_cvref_t<decltype(bio::var_io::default_field_ids)>,
+                                 std::remove_cvref_t<decltype(bio::var_io::field_types_bcf_style<own>)>>;
```

Comment:
Don't now if this follows in a later commit but because of the former commit you can remove the `std::remove_cvref_t` here too, no?

Reply:
Yes and yes :)